instance/api/openedx_appserver.py

```diff
@@ -98,5 +98,5 @@ def make_active(self, request, pk):
             return Response(
                 {"error": "Cannot make an unhealthy app server active."}, status=status.HTTP_400_BAD_REQUEST
             )
-        app_server.instance.set_appserver_active(app_server.pk)
-        return Response({'status': 'App server updated.'})
+        set_appserver_active(app_server.pk)
+        return Response({'status': 'App server activation initiated.'})
```

Comment:
@smarnach Not sure either for the "App server updated" - if I see it again I'll look into it more closely, now I know what I should expect.

> The traceback you included is happening while installing the requirements, but the installation succeeds anyway. I haven't investigated what exactly is causing the traceback, but since pip returns with a success exit code I presume that the error is immaterial.

What does "immaterial" mean? Even if it was not causing an issue (but are we sure this is the case?), I'd rather not leave errors like this in the logs - otherwise we'll just grow used to see traceback there, and start missing issues.

> The log is attached to the load balancer, and not to a particular instance. Since load balancers aren't shown in the UI yet, this means the messages aren't visible in the UI for now. It doesn't make sense to attach the messages to an instance, since the reconfiguration of the load balancer happens for all attached instances at once, so there isn't a single instance to attach it to.

The load balancer is shared by multiple instances/clients, so I'm not sure we will ever want to show it there. And even if we did, it would still make sense to tie logs for actions related to a specific instance to that instance, in addition to the load balancer. 

Could you at least log, at the instance level:
* The output of the `get_load_balancer_configuration()` method
* When this configuration has been applied by the load balancer